apps/cyberstorm-nextjs/middleware.ts

```diff
@@ -24,8 +24,12 @@ export async function middleware(request: NextRequest) {
   const req = request;
   const response = NextResponse.next();
 
+  // What we do here is the following:
```

Comment:
Yes, I can see WHAT it does quite easily in this case. It would be much more valuable to know WHY we do it. Something like "The project root is set to `/cyberstorm` while the NextJs site is served from the same domain as the old Django site. Communities page acts as the front page until we have a proper landing page."

Reply:
I'll literally take your word 👍